pkg/operator/options/options.go

```diff
@@ -48,6 +49,7 @@ func (o *Options) AddFlags(fs *coreoptions.FlagSet) {
 	fs.StringVar(&o.ClusterName, "cluster-name", env.WithDefaultString("CLUSTER_NAME", ""), "[REQUIRED] The kubernetes cluster name for resource discovery.")
 	fs.StringVar(&o.ClusterEndpoint, "cluster-endpoint", env.WithDefaultString("CLUSTER_ENDPOINT", ""), "The external kubernetes cluster endpoint for new nodes to connect with. If not specified, will discover the cluster endpoint using DescribeCluster API.")
 	fs.BoolVarWithEnv(&o.IsolatedVPC, "isolated-vpc", "ISOLATED_VPC", false, "If true, then assume we can't reach AWS services which don't have a VPC endpoint. This also has the effect of disabling look-ups to the AWS on-demand pricing endpoint.")
+	fs.BoolVarWithEnv(&o.IsEKSControlPlane, "is-eks-control-plane", "EKS_CONTROL_PLANE", true, "If true, then assume running on EKS control plane and attempt to discover Kubernetes version via DescribeCluster API ")
```

Comment:
```suggestion
	fs.BoolVarWithEnv(&o.EKSControlPlane, "eks-control-plane", "EKS_CONTROL_PLANE", true, "If true, then assume running on EKS control plane and attempt to discover Kubernetes version via DescribeCluster API ")
```

Let's align our environment variables with our CLI args -- having is in the bool arg feels odd to me